src/utils/migration/constants.ts

```diff
@@ -4,6 +4,7 @@ export const ROOT_COLL_NAME = process.env.MONGO_DB_ROOT_COLL_NAME || "root-domai
 export const SUB_COLL_NAME = process.env.MONGO_DB_SUB_COLL_NAME || "subdomains";
 export const INVALID_COLL_NAME = process.env.MONGO_DB_INVALID_COLL_NAME || "invalid-domains";
 export const INVALID_TX_COLL_NAME = process.env.MONGO_DB_INVALID_COLL_NAME || "invalid-transactions";
+export const INVALID_REVOKES_COLL_NAME = process.env.MONGO_DB_INVALID_COLL_NAME || "invalid-revokes";
```

Comment:
make this check a different `process.env` var first, it shouldnt be looking for the same key as the above line